internal/server/server.go

```diff
@@ -304,3 +320,36 @@ func UpdateServerTags(service *service.Service, serverUUID string, oldTags, newT
 
 	return AddNewServerTags(service, serverUUID, newTags)
 }
+
+// ValidateServerAndStorageBackupRules checks if the server and (to be) attached storages have conflicting backup rules.
+func ValidateServerAndStorageBackupRules(service *service.Service, d *schema.ResourceData) error {
+	if _, ok := d.GetOk("simple_backup"); ok {
+		if attr, ok := d.GetOk("storage_devices"); ok {
+			sds := attr.(*schema.Set).List()
+
+			for _, i := range sds {
+				e := i.(map[string]interface{})
+				id := e["storage"]
+
+				sd, err := service.GetStorageDetails(&request.GetStorageDetailsRequest{
```

Comment:
Doing a request for each storage is imo a bit heavy, but I haven't found a better way to validate this. If you have some other ideas please let me know